README.md

```diff
@@ -3,16 +3,70 @@
 Here we are sharing two assets we have developed todate click on either link to dive in further!
 
 
-1. GNURadio - Azure Out of Tree Module
-2. Azure software radio Developer Virtual Machine [Here](./pages/devvm.md)
+1. [GNURadio - Azure Out of Tree Module](./gr-azure-software-radio/docs/README.md)
+2. [Azure software radio Developer Virtual Machine](./pages/devvm.md)
 
+## Getting Started
+
+To get started, first please follow the install guides below. 
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+GnuRadio 3.9.0 or Greater
+python 3.8 or greater
+```
+
+### Installing Azure software radio OOT
```

Comment:
This seems to be duplicate info of what's in the lower level README - suggest we remove this info from this level and merge with build and test info in ./gr-azure-software-radio/README.md